warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryFieldsVisitor.java

```diff
@@ -156,7 +157,12 @@ public Object visit(ASTIdentifier node, Object data) {
     @Override
     public Object visit(ASTFunctionNode node, Object data) {
         JexlArgumentDescriptor desc = JexlFunctionArgumentDescriptorFactory.F.getArgumentDescriptor(node);
-        Set<String> fields = desc.fields(helper, null);
+        Set<String> fields = null;
+        try {
+            fields = desc.fields(helper, null);
```

Comment:
no null check for desc here?  just asking since i saw you add one in another class.

Reply:
Whoops - good catch, that's an oversight.